key-manager-test/src/test/scala/org/bitcoins/keymanager/WalletStorageTest.scala

```diff
@@ -224,4 +342,33 @@ class WalletStorageTest extends BitcoinSWalletTest with BeforeAndAfterEach {
 
       assert(read == expected)
   }
+
+  it must "write and read an unencrypted ExtPrivateKey from disk" in {
+    walletConf: WalletAppConfig =>
+      assert(!walletConf.seedExists())
+      assert(!walletConf.seedExists())
+      val mnemonicCode = CryptoGenerators.mnemonicCode.sampleSome
+      val writtenMnemonic = DecryptedMnemonic(mnemonicCode, TimeUtil.now)
+      val seedPath = getSeedPath(walletConf)
+      WalletStorage.writeMnemonicToDisk(seedPath, writtenMnemonic)
+
+      val password = getBIP39PasswordOpt().getOrElse(BIP39Seed.EMPTY_PASSWORD)
```

Comment:
This seems very suspicious. What's going on here? This is also used above. Is there a reason we don't just use `None`? And then inside of the `BIP39KeyManager` convert `None -> BIP39Seed.EMPTY_PASSWORD`?

Reply:
This is fixed in #2234